src/sage/rings/semirings/tropical_polynomial.py

```diff
@@ -168,7 +166,7 @@ def roots(self):
         Return the list of all tropical roots of ``self``, counted with
         multiplicity.
 
-        OUTPUT: a list of tropical numbers
+        OUTPUT: A list of tropical numbers
```

Comment:
Please revert this as our style is (usually) starting with a lower case in this instance.